hmpps-kotlin-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/kotlin/health/ProductIdInfoContributor.kt

```diff
@@ -1,11 +1,13 @@
-package uk.gov.justice.digital.hmpps.testappreactive.health
+package uk.gov.justice.hmpps.kotlin.health
 
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.boot.actuate.info.Info
 import org.springframework.boot.actuate.info.InfoContributor
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
 import org.springframework.stereotype.Component
 
 @Component
+@ConditionalOnProperty(prefix = "hmpps", name = ["info.product-id-enabled"], matchIfMissing = true)
 class ProductIdInfoContributor(@Value("\${product-id:default}") private val productId: String) : InfoContributor {
```

Comment:
I see the condition is under the property prefix "hmpps" - should this @Value also be under hmpps? I think it's nice to keep all related properties together.

Reply:
Ah, unless the `PRODUCT_ID` env var is already a convention.